src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    /** Scanner to take in user input. */
+    private Scanner sc;
+
+    /**
+     * Creates a new Ui.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Reads the input from the user.
+     * @return The input from the user.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!